assets/packaging/PackageInfo.tmpl

```diff
@@ -1,5 +1,5 @@
-<pkg-info format-version="2" identifier="{{.organizationName}}.base.pkg" version="{{.version}}" install-location="/" auth="root">
```

Comment:
One could argue that we're now using templated templates. The first templating done during init, the second during packaging. So the extension of the boxed asset files should probably end in `.tmpl.tmpl`.